testing-guidance/workflow-tests/create-a-gem/create-a-gem-workflow-tests.md

```diff
@@ -0,0 +1,21 @@
+| Step 2 - Gem Details                     | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next                                                                                                                | *   All fields can be edited<br>    *   Gem Display Name<br>    *   Gem System Name<br>    *   Gem Summary<br>    *   Requirements<br>    *   License<br>    *   License URL<br>    *   Origin<br>    *   Origin URL<br>    *   Global Gem Tags<br>    *   Optional Global Gem Tag<br>    *   Optional Global Gem Tag<br>    *   User-defined Gem Tags<br>    *   Gem Location<br>    *   Gem Icon Path<br>    *   Documentation URL<br>*   Global Gem Tags field populates with gem fields |
+| Gem Details - Global Gem Tag duplication | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next<br>4.  Attempt to enter the same Global Gem Tag into 2 of the Global Gem Tag fields                            | *   An error displays notifying the user that duplicates cannot be used for global gem tags                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Step 2 - Required Fields not satisfied   | 1.  Launch the Create a Gem experience<br>2.  Select a gem type/template<br>3.  Left click Next<br>4.  On step 2 page, click next without entering information into some of the required fields (marked by \*) | *   User is unable to progress without required fields and a visual indicator suggests what is missing<br>*   Required Fields:<br>    *   Gem Display Name<br>    *   Gem System Name<br>        *   Alpha numeric requirement<br>        *   Unique error for alpha numeric<br>    *   License<br>    *   Global Gem Tag                                                                                                                                                                   |
+| Step 3 - Creator Details                 | 1.  Launch the Create a Gem experience<br>2.  Select a Gem Type/Template<br>3.  Left Click Next<br>4.  Enter Details for Gem<br>5.  Left Click Next                                                            | *   User is unable to progress without required fields and a visual indicator suggests what is missing<br>*   Required Fields:<br>    *   Creator Name<br>    *   Repository URL                                                                                                                                                                                                                                                                                                            |
```

Comment:
Only creator name is required. It's possible the gem is just created for the local drive, so the existing path info would cover that case.